checkov/terraform/checks/resource/aws/EKSPlatformVersion.py

```diff
@@ -24,7 +24,7 @@ def get_inspected_key(self) -> str:
         return "version"
 
     def get_expected_values(self) -> list[Any]:
-        return ["1.22", "1.23", "1.24", "1.25", "1.26"]
+        return ["1.22", "1.23", "1.24", "1.25", "1.26", "1.27"]
```

Comment:
Can you also remove `1.22`, thanks! 😄 
```suggestion
        return ["1.23", "1.24", "1.25", "1.26", "1.27"]
```